strongbox-distribution/src/main/resources/postinstall

```diff
@@ -0,0 +1,3 @@
+groupadd strongbox
+useradd -d /usr/local/strongbox -g strongbox strongbox
```

Comment:
This should not be a regular user but rather a `system` one. Could you please add `--system` here?